src/twisted/application/twist/test/test_twist.py

```diff
@@ -201,3 +201,102 @@ def run(self):
             )
         )
         self.assertEqual(runners[0].runs, 1)
+
+
+
+class CrippledRunner(object):
```

Comment:
1. "Crippled" could be construed as insulting to disabled people, so please don't use the term in the codebase.
2. It's also not a standard technical term; I think you mean "dummy"? http://xunitpatterns.com/Mocks,%20Fakes,%20Stubs%20and%20Dummies.html